bodo/tests/test_df_lib/test_end_to_end.py

```diff
@@ -2575,3 +2575,59 @@ def test_drop_duplicates():
         sort_output=True,
         reset_index=True,
     )
+
+
+def test_uncompilable_map():
+    """Test for maps that can't be compiled."""
+
+    with assert_executed_plan_count(1):
+        df = pd.DataFrame(
+            {
+                "A": pd.array([0, 1] * 100, "Int32"),
+            }
+        )
+
+        def uncompilable(x):
+            # Numba can't compile functions containing imports.
+            return x
+
+        bdf = bd.from_pandas(df)
+        pdf = df.copy()
+
+        bdf["B"] = bdf["A"].map(uncompilable)
+        pdf["B"] = pdf["A"].map(uncompilable)
+    _test_equal(
+        bdf,
+        pdf,
+        check_pandas_types=False,
+        sort_output=True,
+        reset_index=True,
+    )
+
+
+def test_numba_map():
```

Comment:
I tried this test with objmode inside the function and as expected it fails but I don't know of a nice way to check if objmode is present in the function and I tried falling back when the cfunc calls fails but that didn't work.  So, I suggest that we just make a note in the docs that objmode in these UDFs is not supported.